### PR TITLE
Send telegram bug fix

### LINKED ISSUE #2418: Telegram notification python error
Out of the box telegram integration fails. I followed the install instructions closely, so I don't think I'm missing something. 

```
ERROR: Exception in callback make_message.<locals>.on_media_files(<Future finished result=[]>) at /usr/local/lib/python3.9/dist-packages/motioneye/sendtelegram.py:74
handle: <Handle make_message.<locals>.on_media_files(<Future finished result=[]>) at /usr/local/lib/python3.9/dist-packages/motioneye/sendtelegram.py:74>
Traceback (most recent call last):
  File "/usr/lib/python3.9/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/usr/local/lib/python3.9/dist-packages/motioneye/sendtelegram.py", line 78, in on_media_files
    timestamp = time.mktime(moment.timetuple())
AttributeError: 'str' object has no attribute 'timetuple'
```

### PR DESCRIPTION
**Pull request by sirdel**
Fix a few more issues with the telegram script and tided it up, removing old "email" refrences. Should also fix #2418

### CODE REVIEW

File: motioneye/sendtelegram.py [hunk 1]
```diff
@@ -111,7 +102,7 @@ def on_media_files(media_files):
     if not timespan:
         return on_media_files([])
 
-    logging.debug('waiting for pictures to be taken')
+    logging.debug('waiting %ds for pictures to be taken' % float(timespan))
```
Comment (MichaIng) [inline motioneye/sendtelegram.py:105]: Preferring f-strings over %-formatting:
```suggestion
    logging.debug(f'waiting {float(timespan)}s for pictures to be taken')
```

File: motioneye/sendtelegram.py [hunk 2]
```diff
@@ -125,68 +116,50 @@ def on_media_files(media_files):
...
 
     camera_id = motionctl.motion_camera_id_to_camera_id(options.motion_camera_id)
 
-    logging.debug('timespan = %d' % int(options.timespan))
-
     def on_message(message, files):
         try:
-            print(message)
-            logging.info('sending telegram')
+            logging.info('sending telegram : %s' % message)
```
Comment (MichaIng) [inline motioneye/sendtelegram.py:162]:
```suggestion
            logging.info(f'sending telegram : {message}')
```

File: motioneye/sendtelegram.py [hunk 3]
```diff
@@ -125,68 +116,50 @@ def on_media_files(media_files):
...
-        args[7] = 'motionEye on {} <{}>'.format(
-            socket.gethostname(), args[8].split(',')[0]
-        )
-
     options = parse_options(parser, args)
-    print(options)
+    
     meyectl.configure_logging('telegram', options.log_to_file)
-
     logging.debug('hello!')
```
Comment (MichaIng) [inline motioneye/sendtelegram.py:149]: Looks like this debug log was nothing more than "reached here", becoming obsolete with the full options debug log below?
```suggestion
```

File: motioneye/sendtelegram.py [hunk 4]
```diff
@@ -15,27 +15,19 @@
...
 import signal
 import socket
-import sys
 import time
 
 import pycurl
 from tornado.ioloop import IOLoop
 
-from motioneye import config, mediafiles, motionctl, settings, utils
+from motioneye import config, mediafiles, motionctl, settings, utils, meyectl
```
Comment (MichaIng) [inline motioneye/sendtelegram.py:28]: Alphabetical sorting:
```suggestion
from motioneye import config, mediafiles, meyectl, motionctl, settings, utils
```

File: motioneye/sendtelegram.py [hunk 5]
```diff
@@ -125,68 +116,50 @@ def on_media_files(media_files):
...
     fut = utils.cast_future(
         mediafiles.list_media(camera_config, media_type='picture', prefix=prefix)
     )
     fut.add_done_callback(on_media_files)
-
     io_loop.start()
 
 
 def parse_options(parser, args):
+    parser.description='Send Telegram using bot api'
```
Comment (MichaIng) [inline motioneye/sendtelegram.py:130]:
```suggestion
    parser.description = 'Send Telegram using bot api'
```

File: motioneye/sendtelegram.py [hunk 6]
```diff
@@ -125,68 +116,50 @@ def on_media_files(media_files):
...
     parser.add_argument('api', help='telegram api key')
     parser.add_argument('chatid', help='telegram chat room id')
-    parser.add_argument('msg_id', help='the identifier of the message')
     parser.add_argument('motion_camera_id', help='the id of the motion camera')
-    parser.add_argument('moment', help='the moment in ISO-8601 format')
+    parser.add_argument('moment', help='the moment in ISO-8601 format', type=datetime.datetime.fromisoformat)
```
Comment (MichaIng) [inline motioneye/sendtelegram.py:134]: Line length cap enforced by pre-commit black:
```suggestion
    parser.add_argument(
        'moment',
        help='the moment in ISO-8601 format',
        type=datetime.datetime.fromisoformat,
    )
```

File: motioneye/sendtelegram.py [hunk 7]
```diff
@@ -125,68 +116,50 @@ def on_media_files(media_files):
...
-        args = args[:7] + [_from] + args[7:]
-
-    if not args[7]:
-        args[7] = 'motionEye on {} <{}>'.format(
-            socket.gethostname(), args[8].split(',')[0]
-        )
-
     options = parse_options(parser, args)
-    print(options)
+
```
Comment (MichaIng) [inline motioneye/sendtelegram.py:147]:
```suggestion

```

File: motioneye/sendtelegram.py [hunk 8]
```diff
@@ -125,68 +116,50 @@ def on_media_files(media_files):
...
-
     options = parse_options(parser, args)
-    print(options)
+    
     meyectl.configure_logging('telegram', options.log_to_file)
-
     logging.debug('hello!')
-    message = messages.get(options.msg_id)
+    logging.debug(options)
+
```
Comment (MichaIng) [inline motioneye/sendtelegram.py:151]:
```suggestion

```

**Comment by MichaIng** [comment]
Many thanks, great way to solve the `moment` type 👍. Looks good apart of some format suggestions and satisfying CI. I didn't have a closer look at some changes yet, e.g. why you removed `motion_start` from the event line.

**Comment by sirdel** [comment]
I removed `motion_start` because currently there is only one fixed message :-

> `Motion has been detected by camera "%(camera)s/%(hostname)s" at %(moment)s (%(timezone)s).`

it seemed redundant to pass in a parameter that's not used.

**Comment by MichaIng** [comment]
Ah now I see where `motion_start` is used in `sendmail.py`. Makes all sense then 👍.

**Comment by MichaIng** [comment]
Could you please address the suggestions, also to satisfy CI check? So we can merge 🙂.

**Review by sirdel — COMMENTED** [review]
resolved spacing/format suggestions

**Review by MichaIng — APPROVED** [review]

**Pull request merged**